Overview.src.html

```diff
@@ -835,6 +847,147 @@ <h2 class="no-num no-toc">
       </section>
       <section>
         <h3>
+          Interface <code>Availability</code>
+        </h3>
+        <pre class="idl">
+interface <dfn>Availability</dfn> : EventTarget {
+  readonly attribute boolean value;
+  attribute <span data-anolis-spec=
+"w3c-html">EventHandler</span> <span>onchange</span>;  
+};
+</pre>
+        <p>
+          The <code>change</code> event is fired during the execution of the
+          <span>monitoring presentation display availability</span> algorithm
+          when the <dfn>presentation display availability</dfn> changes. It is
+          fired at the <span><code>Availability</code></span> object, using the
+          <code>ChangeEvent</code> interface. The <code>Availability</code>
```

Comment:
Ideally we would add this document as a reference here.  This can be done in a subsequent PR.